Bindings/Java/Matlab/Utilities/osimC3D.m

```diff
@@ -170,10 +196,6 @@ function writeMOT(obj,varargin)
         % osimC3d.writeMOT()                       Write to dir of input c3d.
         % osimC3d.writeMOT('Walking.mot')          Write to dir of input c3d with defined file name.
         % osimC3d.writeMOT('C:/data/Walking.mot')  Write to defined path input path.
-        % 
```

Comment:
Can we remove these comments unless you use for debugging, I was worried about sneaking absolute paths into user facing code.

Reply:
They are actually help statements for the function that a user can call;

``` java
>> help osimC3D/writeMot
  Write force data to mot file.
  osimC3d.writeMOT()                       Write to dir of input c3d.
  osimC3d.writeMOT('Walking.mot')          Write to dir of input c3d with defined file name.
  osimC3d.writeMOT('C:/data/Walking.mot')  Write to defined path input path.
  
  This function assumes point and torque data are in mm and Nmm and
  converts them to m and Nm. If your C3D is already in M and Nm,
  comment out the internal function convertMillimeters2Meters()
```
In this case, you can use either of the three methods (empty, filename, or full path) to write the output.